replica/replica.go

```diff
@@ -738,6 +738,8 @@ func (r *Replica) encodeToFile(obj interface{}, file string) error {
 	}
 	defer f.Close()
 
+	// crash at this point leads to stale .tmp file
```

Comment:
this comment may not be required as os.Create seems to be truncating the file if already exists..
can you please verify with a test program before adding any cleanup code here?

Reply:
@vishnuitta i have verified .tmp file will remain, if replica crashes at this point. @payes asked me to put this comment.